mos-platform/geos-cbm/geos_constants.h

```diff
@@ -0,0 +1,752 @@
+/***********************************************************************
+ * geos_constants.h
+ *
+ * This file contains #defines for use in GEOS applications
+ * with llvm-mos C compiler.
+ *
+ * Transcribed from geoProgrammer manual by DiSc.
+ *
+ ***********************************************************************/
+
+#ifndef GEOS_CONSTANTS_H
+#define GEOS_CONSTANTS_H
+
+/***********************************************************************
+ * Miscellaneous defines
+ ***********************************************************************/
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define LOW_BYTE(x) (*((unsigned char *)&(##x) + 0))
```

Comment:
This could use a comment describing what's legal to pass to it; it doesn't look likely that these will work on arbitrary C expressions.

Reply:
Not needed anymore. I removed it.